src/app/econ/supply-demand/lesson/SupplyDemandLesson.tsx

```diff
@@ -0,0 +1,807 @@
+            </li>
+            <li>
+              <Collapsible
+                ExampleContent={
+                  <>
+                    <Emphasize>
+                      &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;👉 Example:{" "}
+                    </Emphasize>{" "}
+                    If the cost of potatoes go up, that will cause the supply of
+                    french fries to go down. <br />
+                    <Emphasize>
+                      &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;👉 Notice:
+                    </Emphasize>{" "}
+                    A decrease in supply will shift the supply curve to the
+                    left.
+                    <br /> <br />
+                  </>
+                }
+              >
+                Input Prices (cost of raw materials).
+              </Collapsible>
+            </li>
+            <li>
+              <Collapsible
+                ExampleContent={
+                  <>
+                    <Emphasize>
+                      &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;👉 Example:{" "}
+                    </Emphasize>{" "}
+                    If there are machines that make french fries faster and
+                    cheaper, it would cause the supply of french fries to go up.
+                    <br />
+                    <Emphasize>
+                      &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;👉 Notice:
+                    </Emphasize>{" "}
+                    An increase in supply will shift the supply curve to the
+                    right. <br /> <br />
+                  </>
+                }
+              >
+                Technology (production efficiency)
+              </Collapsible>
+            </li>
+            <li>
+              <Collapsible
+                ExampleContent={
+                  <>
+                    <Emphasize>
+                      &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;👉 Example:{" "}
+                    </Emphasize>{" "}
+                    If there are more french fry sellers, it would cause the
+                    supply of french fries to go up. <br />
+                    <Emphasize>
+                      &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;👉 Example:
+                    </Emphasize>{" "}
+                    If there are less french fry sellers, it would cause the
+                    supply of french fries to go down. <br /> <br />
+                  </>
+                }
+              >
+                Number of Sellers.
+              </Collapsible>
+            </li>
+            <li>
+              <Collapsible
+                ExampleContent={
+                  <>
+                    <Emphasize>
+                      &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;👉 Example:{" "}
+                    </Emphasize>{" "}
+                    If people expect the price of french fries to go up, they
+                    will hold back on selling them now and wait for the price to
+                    go up. <br />
+                    <br />
```

Comment:
nit: This might be personal preference, but 2 br tags seems like too much space. You can just add margin-bottom to the collapsible component